app/support/reservations/date_support.rb

```diff
@@ -11,7 +11,7 @@ module Reservations::DateSupport
                 :actual_end_date, :actual_end_hour, :actual_end_min, :actual_end_meridian,
                 :actual_duration_mins
 
-    before_validation :set_all_split_times
+    before_validation :set_all_split_times, if: :end_at_required?
```

Comment:
I get that we need it in certain cases, but could we name it something else? I'd be fine being an alias, but this feels confusing as to why it's here. I'd be ok if it's just an alias.